crates/kernel/testsuite/moot/math.moot

```diff
@@ -0,0 +1,84 @@
+// Adapted from https://github.com/toddsundsted/stunt/blob/a4158f5835f1beb9d754c92fd5b3a137e459aabf/test/test_math.rb
+@programmer
+
+// test_that_random_0_is_invalid
+; random(0);
+E_INVARG
+
+; random(42.123);
+E_TYPE
+
+// test_that_random_1_returns_1
+; return random(1);
+1
+
+// test_that_random_returns_a_number_between_1_and_2147483647
+; for i in [1..1000]; n = random(); if (n <= 0 || n > 2147483647); return "FAIL"; endif; endfor; return "PASS";
+"PASS"
+
+// test_that_random_requires_a_positive_integer
+; random(-3);
+E_INVARG
+
+// more than 1 argument is invalid
+; random(1, 2, 3);
+E_ARGS
+
+// test_that_division_by_zero_fails
+; return 1.1 / 0.0;
+E_DIV
+; 1 / 0;
+E_DIV
+; 1.1 % 0.0;
+E_DIV
+; 1 % 0;
+E_DIV
+
+// test_the_minint_edge_case
```

Comment:
we're 64-bit instead of 32-bit ints, unlike toast, so it's not even minint in our case

Reply:
Right. I guess there's a question of: do we consider "we use 32 bit ints" part of "the MOO API"? And I'd say: probably not. Anything that depends on this distinction deserves to break :D

I'll extend the test to the `i64::MIN` case.